client/src/graphs/svg/renderers/EpiRenderer.ts

```diff
@@ -32,11 +35,15 @@ export default class EpiRenderer extends SVGRenderer {
   }
 
   static calcNodeColor (d: d3.Selection<any, any, any, any>): string {
-    if ((d as any).nodes) {
-      return Colors.NODES.CONTAINER;
-    } else {
-      return (d as any).data.role?.includes(NodeTypes.NODES.VARIABLE) ? Colors.NODES.VARIABLE : DEFAULT_STYLE.node.fill;
-    }
+    return (d as any).nodes ? Colors.NODES.CONTAINER : DEFAULT_STYLE.node.fill;
+  }
+
+  static calcNodeStrokeStyle (d: d3.Selection<any, any, any, any>): string {
+    return !(d as any).nodes && !(d as any).data.role?.includes(NodeTypes.NODES.VARIABLE) ? '5,5' : null;
+  }
+
+  static calcNodeStrokeWidth (d: d3.Selection<any, any, any, any>): number {
+    return !(d as any).nodes && !(d as any).data.role?.includes(NodeTypes.NODES.VARIABLE) ? DEFAULT_STYLE.node.strokeWidth + 1 : DEFAULT_STYLE.node.strokeWidth;
```

Comment:
This line might be more readible on multiple lines.

Reply:
Addressed.